### DISCUSSION TITLE
Ninja 2.6.10

### DISCUSSION THREAD

**Discussion by szapp** [szapp/Ninja#25] [General]
This version supports the [latest changes of LeGo](https://github.com/Lehona/LeGo/compare/f4cb515eda5028abb0e0baeafc4ead682bc7bc45...a7db05db2e8a2d0bd91aee84d4b15b904ae983f5) and safer handling of non-persistent NPC and VDF verification.

- [Change log](../wiki/Changelog#v2610-apr-24-2021)
- [Installation instructions](../wiki/Installation-(EN)#wiki-wrapper)
- [Check sums](../wiki/Checksums#wiki-wrapper)

<hr /><em>This discussion was created from the release <a href='https://github.com/szapp/Ninja/releases/tag/v2.6.10'>Ninja 2.6.10</a>.</em>

**Comment by ghost** [comment]
How to install Ninja. I got Gothic from GOG (1.08k hotfix) with installed Union 1.0k + Player Kit 2.8. I can not see version info (G1CP 1.1) when i'm in game menu. How to Fix IT ? Thanks.

**Reply by szapp** [reply]
Have you checked the [installation instructions](https://github.com/szapp/Ninja/wiki/Installation-(EN)#wiki-wrapper)? The remaining step after what you have done is to install Ninja through the [provided setup](https://github.com/szapp/Ninja/releases), ideally with administrative privileges.

BTW: Why Union 1.0k and not the latest?

**Reply by ghost** [reply]
1.0k is newest version downloaded from Union Resource Manager.

**Reply by szapp** [reply]
Ah my bad, I had confused the version numbers. Thanks.

**Reply by ghost** [reply]
So how to install it? What am doing wrong ?

**Reply by szapp** [reply]
Well, did you download and install `Ninja-2.6.10.exe` as I described?

**Reply by ghost** [reply]
Of course. Ninja-2.6.10.exe downloaded and installed from your Github.

**Reply by szapp** [reply]
You can confirm your installation with these [troubleshooting instructions](https://github.com/szapp/Ninja/wiki/Troubleshooting-%28EN%29#wiki-wrapper).

**Comment by SirPotatorito** [comment]
I have the same problem as the guy above, although I own the Steam version, I installed both Union and The Community Patch, but when it came to install Ninja, the files seem to go in the DIR file, but for some reason when I type in the console command "Ninja" its says that it's an unknown command, so I can't install the C-Patch, any ideas? I have seen a fix where I reinstall everything, but it also doesn't help me, thanks in advance really appreciate the work.

**Reply by szapp** [reply]
> the files seem to go in the DIR file

What do you mean by that exactly? What is a DIR file?

Ninja is offered with an executable setup, that - given administrative privileges - will install Ninja without any additional manual steps.

**Reply by SirPotatorito** [reply]
I meant as in the Directory file (The installation path), it follows it correctly as I can see some "Ninja" files, but it just doesn't work and as I said I tried to confirm it using the method provided by typing "Ninja" in the in-game console only for the game to not recognize it, and I did install the latest version from your GitHub directly, could it be Union's fault? I really don't know, would you recommend using the Union resource manager to install it? I hope that give an idea on the situation, thanks again for your work

**Reply by SirPotatorito** [reply]
Okay, sorry for the mess, all I had to do was to add Ninja.dll** to the Plugins list in SystemPack.ini, after that it loaded perfectly, thanks for your time and work of course.

**Reply by szapp** [reply]
No worries, I see how that was frustrating. I am curious. If Ninja is installed **last** (with the setup provided here) it should make the necessary entries automatically - irrespective of whether Union, SystemPack or neither is installed.
